projects/igniteui-angular/src/lib/core/styles/components/navdrawer/_navdrawer-theme.scss

```diff
@@ -305,11 +305,7 @@
         }
 
         // Need this to override the igx-buttons
-        &%igx-button--flat,
-        &%igx-button--outlined,
-        &%igx-button--raised,
-        &%igx-button--fab,
-        &%igx-button--icon  {
+        &igx-button {
```

Comment:
Hm, this is not exactly as requested :( I mean to set it as the proper selector `[igxButton]` as igxButton is a directive, not a component.